src/stream/successors.rs

```diff
@@ -58,31 +63,19 @@ where
 /// # }) }
 ///
 /// ```
-pub fn successors<F, Fut, T>(start: Option<T>, func: F) -> Successors<F, Fut, T>
+pub fn successors<F, Fut, T>(first: Option<T>, succ: F) -> Successors<F, Fut, T>
```

Comment:
Can we perhaps mark this function as "unstable"?

Reply:
I think I did this now? I am not sure if I marked all the spots?